src/PowerShellEditorServices.Protocol/Server/LanguageServerSettings.cs

```diff
@@ -292,7 +284,25 @@ private Hashtable GetCustomPSSASettingsHashtable(int tabSize, bool insertSpaces)
                     {"PSUseCorrectCasing", new Hashtable {
                         {"Enable", UseCorrectCasing}
                     }},
-                }}
+                };
+            if (AutoCorrectAliases)
+            {
+                ruleConfigurations.Add("PSAvoidUsingCmdletAliases", new Hashtable());
```

Comment:
The current pattern seems to always include the rule entry and set `Enable` to the boolean given. Is there a reason that shouldn't be done here?

Also, does this need to set `Enable = true` in the hashtable body?

Reply:
The `PSAvoidUsingCmdletAliases` is a script analysis rule that inherits only from `IScriptRule` (see [here](https://github.com/PowerShell/PSScriptAnalyzer/blob/master/Rules/AvoidAlias.cs)). Formatting rules typically inherit from [ConfigurableRule](https://github.com/PowerShell/PSScriptAnalyzer/blob/master/Engine/Generic/ConfigurableRule.cs), which is what makes them have the `Enable` property. This is just the API of this specific rule, therefore although it would be nice to have them all consistent, I don't think we could even change that in PSSA without breaking people that use the `PSAvoidUsingCmdletAliases` for code analysis in their setting files.